packages/flutter/lib/src/material/typography.dart

```diff
@@ -69,11 +110,30 @@ class TextTheme {
       TextStyle.lerp(begin.button, end.button, t)
     );
   }
-
 }
 
+/// The two material design text themes.
+///
+/// [Typography.black] and [Typography.white] define the two text themes used in
+/// material design. The black text theme, which uses darkly colored glyphs, is
+/// used on lightly colored backgrounds in light themes. The white text theme,
+/// which uses lightly colored glyphs, is used on darkly colored backgrounds in
+/// in light themes and in dark themes.
+///
+/// To obtain the current text theme, call [Theme.of] with the current
+/// [BuildContext] and read the [ThemeData.textTheme] property.
+///
+/// See also:
+///
+///  * [Theme]
+///  * [ThemeData]
+///  * <http://www.google.com/design/spec/style/typography.html>
 class Typography {
   Typography._();
+
+  /// A material design text theme with darkly colored glyphs.
   static const TextTheme black = const TextTheme._black();
+
+  /// A material design text theme with lightly colored glyphs.
```

Comment:
lightly-colored.

But "light" is probably sufficient. In particular, "lightly-colored" means "has a little bit of color" which probably isn't what we mean here.